src/routers/webhooks.py

```
@@ -0,0 +1,91 @@
+    if (
+        not jwt_params.ad
+        and jwt_params.w is not None
+        and jwt_params.w != webhook.workspace_name
+    ):
+        raise AuthenticationException("Unauthorized access to resource")
+
+    return await crud.get_or_create_webhook_endpoint(db, webhook)
+
+
+@router.get("", response_model=Page[schemas.WebhookEndpoint])
+async def list_webhook_endpoints(
```

Ruff (0.12.2)

Missing return type annotation for public function `list_webhook_endpoints`

(ANN201)